### DISCUSSION TITLE
help needed

### DISCUSSION THREAD

**Discussion by abushak** [mskocik/svelecte#52] [Show and tell]
Have a good time of the day or night!

is it possible to ask for a help with your beatiful lib?
sure for $

**Comment by mskocik** [comment]
I am willing to help. What do you need? If you want to support me, you can consider sponsoring on GitHub.

**Reply by abushak** [reply]
i'am trying with custom Svelecte element achieve dependable selects,

By the way is it necessary to make a custom element to achieve dependable select?

i can't resolve issue with __vite_ssr_import_1__.registerSvelecte is not a function

I can become a github sponsor but i saw only 50 usd as one time/monthly payment. I think it should cost more.

Anyway i can show and discuss what exactly we need with you more by email: ihor.kashuba@gmail.com, google meet or any other way you prefer
 
Is it possible?

**Reply by mskocik** [reply]
> i'am trying with custom Svelecte element achieve dependable selects,
> 
> By the way is it necessary to make a custom element to achieve dependable select?
> 
> i can't resolve issue with **vite_ssr_import_1**.registerSvelecte is not a function
> 
> I can become a github sponsor but i saw only 50 usd as one time/monthly payment. I think it should cost more.

Probably it could be higher, but I think this is more likely for someone to pay/sponsor.
 
> Anyway i can show and discuss what exactly we need with you more by email: [ihor.kashuba@gmail.com](mailto:ihor.kashuba@gmail.com), google meet or any other way you prefer
> 
> Is it possible?

I will contact you through email.
